kubernetes_yaml/model_dowloader/job/download_job.yaml

```diff
@@ -0,0 +1,40 @@
+## Kubernetes job to download the model
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: download-job
+  namespace: default
+spec:
+  template:
+    spec:
+      containers:
+      - name: download
+        image: quay.io/rcook/model-download:latest
+        volumeMounts:
+        - name: model
+          mountPath: /opt/models
+        env:
+        - name: HF_TOKEN
+          valueFrom:
+            secretKeyRef:
+              name: my-secret
+              key: huggingtoken
+        - name: AWS_BUCKET_NAME
+          value: "model-store-def7b8ef-55e6-4042-9faa-176bf68f8523"
+        - name: AWS_ACCESS_KEY
+          valueFrom:
+            secretKeyRef:
+              name: model-store
+              key: AWS_ACCESS_KEY_ID
+        - name: AWS_SECRET_KEY
+          valueFrom:
+            secretKeyRef:
+              name: model-store
+              key: AWS_SECRET_ACCESS_KEY
+        - name: S3_ENDPOINT
```

Comment:
we should also add another env var for the `S3_FOLDER` which is being used here: https://github.com/redhat-et/ilab-on-ocp/blob/downloader/kubernetes_yaml/model_dowloader/container_file/download_hf_model.py#L69